bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml

```diff
@@ -61,6 +66,30 @@ Deploy Cumulus buckets and Cumulus and Orca modules:
         #!/bin/bash
         . ./integration_test/shared/setup-orca.sh
         bash integration_test/deployment-orca.sh
+
+#job for running ingest integration tests
+Run Ingest Integration tests:
+  key: RIIT
+  other:
+    clean-working-dir: true
+  docker:
+    image: maven.earthdata.nasa.gov/cumulus_orca
+    volumes:
+      ${bamboo.working.directory}: ${bamboo.working.directory}
+      ${bamboo.tmp.directory}: ${bamboo.tmp.directory}
+  tasks:
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+        . ./integration_test/shared/setup-orca.sh
+        cd integration_test/workflow_tests
+        bash bin/run_tests.sh
+
```

Comment:
Perhaps a bit of an overload for the bucket creation script. How troubling would it be to move integration tests to their own .yaml file?

Reply:
We can create a new bamboo pipeline only to run integration tests if we want but that means we have to manually run the plan once deployment is complete.

Reply:
Missed that this actually handles deploying Cumulus AND ORCA AND buckets. Thought it read as "Deploy Cumulus/ORCA buckets. The separate stages are fine.